src/libs/OptionsListUtils.js

```diff
@@ -56,19 +56,23 @@ function getPersonalDetailsForLogins(logins, personalDetails) {
  * @return {String}
  */
 function getSearchText(report, personalDetailList) {
-    const searchTerms = [];
+    // Use a set because it skips adding duplicates
+    // which could potentially significantly speed up our
+    // regex search if someone has thousands of chats
```

Comment:
weird line breaks here, the max is 120 these look much shorter.

Reply:
Will fix!